docs/integrate/visualize.md

```diff
@@ -63,7 +63,7 @@ based on Apache Superset.
 
 ```{div}
 :style: "float: right"
-[![](https://www.cluvio.com/images/logos/cluvio/cluvio-logo-full_color-on_dark.svg){w=180px}](https://www.cluvio.com/)
+[![cluvio-logo-full_color-on_dark.svg](https://github.com/crate/crate-clients-tools/assets/453543/cac142ef-412a-4a67-a63f-bf9d1ce92c84){w=180px}](https://www.cluvio.com/)
```

Comment:
I am just using GitHub as image host here. It worked well for a while already, so I never looked back.

/cc @matkuliak